Client/src/Components/Charts/Gauge/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { RadialBarChart, RadialBar, ResponsiveContainer } from "recharts";
+import PropTypes from "prop-types";
+
+const Gauge = ({ progressValue, displayText, theme }) => {
+	const BASE_TEXT_STYLE = {
+		textAnchor: "middle",
+		dominantBaseline: "middle",
+	};
+	const getStyles = (theme) => ({
+		text: {
+			...BASE_TEXT_STYLE,
+			style: { fontSize: "11px", fontWeight: 400, fill: theme.palette.text.tertiary				
+			 },
+		},
+	});
+	const data = [{ value: 100 }];
+	const commonRadialBarProps = {
+		minAngle: 15,
+		clockWise: true,
+		dataKey: "value",
+		cornerRadius: 0
+	};
+	return (
+		<ResponsiveContainer
+			width="100%"
+			height={150}
```

Comment:
Might be better to leave this fully responsive and leave the height up to the parent?

Reply:
Yes, leave then responsive to the parent, resolved in : https://github.com/bluewave-labs/bluewave-uptime/pull/1087/commits/5e54f5c75763e49a3efb630e7baabf71a483f0c1